wpilibNewCommands/src/main/native/include/frc2/command/CommandHelper.h

```diff
@@ -24,17 +24,17 @@ namespace frc2 {
 template <typename Base, typename CRTP,
           typename = std::enable_if_t<std::is_base_of_v<Command, Base>>>
 class CommandHelper : public Base {
-  using Base::Base;
-
  public:
-  CommandHelper() = default;
+  virtual ~CommandHelper() = default;
```

Comment:
What's the reason for this?  Do we ever destroy an object via `CommandHelper*`?

Reply:
Not sure, I'm trying to figure out what's causing the memory leak asan is finding.
Given that `CommandHelper` is a base class, shouldn't it have a virtual dtor?

Reply:
So any idea what could be causing the memory leak?